liquibase-standard/src/main/java/liquibase/snapshot/jvm/IndexSnapshotGenerator.java

```diff
@@ -83,9 +83,12 @@ protected void addTo(DatabaseObject foundObject, DatabaseSnapshot snapshot) thro
                     }
                     Boolean descending = "D".equals(ascOrDesc) ? Boolean.TRUE : ("A".equals(ascOrDesc) ? Boolean
                             .FALSE : null);
-                    index.addColumn(new Column(row.getString("COLUMN_NAME")).setComputed(false).setDescending(descending).setRelation(index.getRelation()));
+                    Boolean included = row.getBoolean("IS_INCLUDED_COLUMN");
+                    Column c = new Column(row.getString("COLUMN_NAME")).setComputed(false).setDescending(descending).setRelation(index.getRelation()).setIncluded(included);
```

Comment:
Similar here. I'm not sure what could return `row.getBoolean("IS_INCLUDED_COLUMN")` for a database that doesn't support included columns, so I think it would safer if we surround this piece of code into a similar `IF` block.

Reply:
done.